environments/skeleton/{{cookiecutter.environment}}/terraform/compute.tf

```diff
@@ -25,4 +25,6 @@ module "compute" {
   k3s_token = var.k3s_token
   control_address = [for n in openstack_compute_instance_v2.control["control"].network: n.fixed_ip_v4 if n.access_network][0]
   security_group_ids = [for o in data.openstack_networking_secgroup_v2.nonlogin: o.id]
+
+  compute_init_enable = each.value.compute_init_enable
```

Comment:
Don't we need to default to the empty list on this? Otherwise we *always* have to supply it?